src/mode/modeNormal.ts

```diff
@@ -36,6 +39,56 @@ export class NormalMode extends Mode {
         "%" : async () => { return vscode.commands.executeCommand("editor.action.jumpToBracket"); },
         ">>" : async () => { return vscode.commands.executeCommand("editor.action.indentLines"); },
         "<<" : async () => { return vscode.commands.executeCommand("editor.action.outdentLines"); },
+        "cw" : async (m) => {

```

Comment:
Can we reuse the `changeoperator`? https://github.com/VSCodeVim/Vim/blob/master/src/operator/change.ts